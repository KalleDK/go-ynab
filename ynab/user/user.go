package user

type User struct {
	ID ID `json:"id"`
}
