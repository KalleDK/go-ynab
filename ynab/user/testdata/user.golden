{
    "id": "a5489693-ab4d-371c-bc95-7156b3b41be7"
}